src/EnergyPlus/SetPointManager.cc

```diff
@@ -2658,10 +2673,6 @@ namespace SetPointManager {
 
 		}
 
-		if ( ErrorsFound ) {
-			ShowFatalError( RoutineName + "Errors found in input.  Program terminates." );
-		}
-
```

Comment:
This fatal in the middle of SPM GetInput is not standard practice. There are 2 possibilities, 1) it was simply left over and not removed when the code was changed to only fatal after all SPMs were read in, or 2) it's here to protect a crash. After review, I see nothing below this point that would cause a crash.